### DISCUSSION TITLE
Welcome to discussions

### DISCUSSION THREAD

**Discussion by samsp-msft** [dotnet/yarp#180] [General] [answered]
Discussions are a new beta feature for github we are trying out. Its a less formal place to ask questions, and discuss the project without having to express it in terms of an issue.
We all can learn something from each other, so please be open minded and respectful of other community members. If there are questions you have about YARP, or reverse proxy scenarios, here is a good place to ask them.

**Comment by karelz** [comment, marked as answer]
Triage: 2 years old, let's mark it answered ;)
